sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -468,6 +469,7 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
             component: InstitutionStudentProfile,
             meta: {
               clientType: ClientIdType.Institution,
+              allowOnlyBCPublic: true,
```

Comment:
adding `allowOnlyBCPublic` to the parent (line 457) will it protect all children? As discussed, you can just try and check

Reply:
I tested by using allowOnlyBCPublic in parent onluy and it works perfectly. thanks.

Reply:
Thanks for trying it. As discussed, we can test and remove the other meta from the children in another ticket